charts/kube-state-metrics/templates/rolebinding.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if and (eq  .Values.rbac.create true) (eq .Values.rbac.useClusterRole false) -}}
+{{- range (split "," $.Values.namespace) }}
```

Comment:
the range makes sense over here. I see that the `{{- end -}}` is placed correctly. I am just confused by the range block used in role.yaml

Reply:
The range block in role - is because a Clusterrole must be ONLY once.. But when operating on only a few namespaces - each must have the relevant role.. so there must be a role per namespace we operate on (as it's NOT a ClusterRole - it only affects the individual namespace and lives within it).
I hope that makes it clearer :)

Reply:
and there USED to be a clusterrole.yaml and a role.yaml- but I was asked to merge the two - as they were 99% identical.